src/applications/personalization/appointments/actions/index.js

```diff
@@ -108,28 +108,28 @@ export function fetchConfirmedFutureAppointments() {
 
         // This catches partial errors on the meta object
         if (
-          vaAppointmentsResponse?.meta?.errors?.length > 0 ||
```

Comment:
it's a minor pet peeve of mine when optional chaining is used too aggressively...